### ISSUE TITLE
Too low contrast

### ISSUE THREAD

**Issue by torgeirl** [devilry/trix2#116]
Success Criteria 1.4.3 in WCAG 2 [requires a minimum of color contrast](https://dequeuniversity.com/rules/axe/3.1/color-contrast?application=lighthouse). After upgrading Django (#102) and making encreased contrast default (#109) we need to make sure we provide a sufficient color contrast.

Some issues before addressing #102 and #109:
* Element has insufficient color contrast of 1.94 (foreground color: #ffffff, background color: #f0ad4e, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1 `<a href="/authenticate/login/?next=/" class="btn btn-warning"> <span class="fa fa-sign-in"></span> Logg inn </a>`
* Element has insufficient color contrast of 4.32 (foreground color: #777777, background color: #fbfbfb, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1 `<p class="text-muted">Course name</p>` (frontpage course headlines)
